backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/keyboardheight/AndroidRKeyboardHeightProvider.java

```diff
@@ -4,14 +4,16 @@
 import android.annotation.SuppressLint;
 import android.app.Activity;
 import android.content.res.Configuration;
+import android.os.Build.VERSION_CODES;
 import android.view.View;
-import androidx.core.graphics.Insets;
-import androidx.core.view.OnApplyWindowInsetsListener;
-import androidx.core.view.ViewCompat;
-import androidx.core.view.WindowInsetsCompat;
+import android.view.View.OnApplyWindowInsetsListener;
+import android.view.WindowInsets;
+import android.graphics.Insets;
+import androidx.annotation.RequiresApi;
```

Comment:
you can replace `import androidx.annotation.RequiresApi` with `import android.annotation.RequiresApi;`